### PR TITLE
EOS-22834: Adopt resource-type change from fru to hw

### PR DESCRIPTION
**Pull request by srikant03**
# Backend

# Problem Statement
EOS-22834: Adopt resource-type change from fru to hw, where ever applicable.

- _Overview: JIRA number/GitHub Issue added to PR: [Y/N]:_ Y
- _Describe the problem this patch intends to solve._ We need to adopt SSPL resource_type field change from 'fru' to 'hw' in csm side.

## Design

- _For Bug describe the fix here._
- _For feature, post the link to the solution page._
- _Any northbound interface change and has been notified to other gatekeepers [Y/N]:_

## Coding

- _Coding conventions are followed and code is consistent. Yes/No?_ Yes
- _Confirm All CODACY errors are resolved. Yes/No?_ Yes

## Testing

- _Confirm that Test Cases are added (for both the cases, fix and feature). Yes/No?_ No
- _Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability. Yes/No?_ Yes
- _Confirm Testing was performed with installed RPM. Yes/No?_ Yes
- _Confirm all the ut/regression/smoke test has been performed [Y/N]_ Y

## Checklist

- _PR is self-reviewed? Yes/No?_ Yes
- _GitHub Issue is updated_ Yes
- _Jira is updated_
  -  _Check if the description is clear and explained._ 
    -  _Check Acceptance Criterion is defined._
      -  _All the tests performed should be mentioned before Resolving a JIRA._
        -  _Verification needs to be done before marked as Closed/Verified_ 
        - _Any interface change Yes/No?_ No
        - _Change notified to other gatekeepers: Yes/No?_
        - _Code reviews done and addressed: Yes/No?_
        - _Codacy issues reviewed and addressed: Yes/No?_
        - _Deployment test : Done/Not?_ Done
        - _UT/smoke test: Done/Not?_ Done
        - _Jira number added to PR: Yes/No?_ Yes
        - _Github merge done using UI: Yes/No?_
        - _Side effects on other features - deployment/update/node-replacement_
        - _Changes to quick-start-guide/community impact_
        - _All dependent component code PR are raised or already merged Yes/No_ Yes
        - _All dependent code already merged in landing branch Yes/No_ No

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: csm/core/repositories/alerts.py [hunk 1]
```diff
@@ -123,7 +123,7 @@ async def count_by_range(self, time_range: DateTimeRange):
     #         "severity": 1,
     #         "state": "fault_resolved",
     #         "extended_info": {
-    #             "resource_type": "fru",
+    #             "resource_type": "hw",
```
Comment (pawankumarsrivastava) [inline csm/core/repositories/alerts.py:126]: You have made changes to a commented code. Instead remove the commented code

**Review by pawankumarsrivastava — CHANGES_REQUESTED** [review]
There are few files which are not required now and are obsolete.  
test/test_data/health_schema.json 
test/test_data/input.text

Remove these files and all there references from the code.

You can keep experiments/validator_comparison/schema/test_sspl.json as it was an experiment for validations

File: csm/core/repositories/alerts.py (hunk 1 above)
Comment (srikant03) [reply csm/core/repositories/alerts.py:126]: The commented code written for **to dump the data while starting the server** even we can delete that commented code?

Comment (pawankumarsrivastava) [reply csm/core/repositories/alerts.py:126]: Remove the commented code

**Comment by srikant03** [comment]
> There are few files which are not required now and are obsolete.
> test/test_data/health_schema.json
> test/test_data/input.text
> 
> Remove these files and all there references from the code.
> 
> You can keep experiments/validator_comparison/schema/test_sspl.json as it was an experiment for validations

Addressed Comments

**Review by pawankumarsrivastava — CHANGES_REQUESTED** [review]
You have deleted 2 files and have not taken care where they are being used.

**Comment by srikant03** [comment]
> You have deleted 2 files and have not taken care where they are being used.

That files are not used any where, have checked once again.

**Review by pawankumarsrivastava — COMMENTED** [review]
Please add screenshots of the testing done. Also fill in the templates

**Comment by srikant03** [comment]
> Please add screenshots of the testing done. Also fill in the templates

Addressed review comment.

alert_enclosure_serial
alert_psu
alert_fan
alert_disk

**Review by pawankumarsrivastava — APPROVED** [review]

**Review by kupranay — APPROVED** [review]

**Pull request merged**